collie/model/hybrid_pretrained_matrix_factorization.py

```diff
@@ -323,7 +445,11 @@ def save_model(self,
                 raise ValueError(f'Data exists in ``path`` at {path} and ``overwrite`` is False.')
 
         Path(path).mkdir(parents=True, exist_ok=True)
-        joblib.dump(self.item_metadata, os.path.join(path, 'metadata.pkl'))
+        if self.item_metadata is not None:
```

Comment:
Should we update the docstring above like for the other hybrid model's `save_model` method?

Reply:
👍